src/index.html

```diff
@@ -9,6 +9,94 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header
+    class="header"
+    >
+      <a
+      href="#"
+      class="logo-link"
+      >
+        <img
+        src="/src/images/logo.png"
+        alt="Logo">
```

Comment:
[Checklist](https://github.com/Rusik-best86/layout_moyo-header/blob/develop/checklist.md):
- alt attribute should describe the image content
![image](https://github.com/mate-academy/layout_moyo-header/assets/63243136/137c6759-8320-4a63-a31f-0e4527872ee9)